site/data/examples.yml

```diff
@@ -14,18 +14,28 @@
     - name: Webpack
       description: "Import and bundle Bootstrap's source Sass and JavaScript with Webpack."
       url: /examples/tree/main/webpack
-      htmlIndexLocation: src
+      indexPath: src/index.html
     - name: Parcel
       description: "Import and bundle Bootstrap's source Sass and JavaScript via Parcel."
       url: /examples/tree/main/parcel
-      htmlIndexLocation: src
+      indexPath: src/index.html
     - name: Vite
       description: "Import and bundle Bootstrap's source Sass and JavaScript with Vite."
       url: /examples/tree/main/vite
-      htmlIndexLocation: src
+      indexPath: src/index.html
     - name: Bootstrap Icons
       description: "Import and compile Bootstrap's Sass with Stylelint, PurgeCSS, and the Bootstrap Icons web font."
       url: /examples/tree/main/icons-font
+    - name: Bootstrap color modes
+      description: "Import and compile Bootstrap's Sass with Stylelint, and the Bootstrap color modes."
+      url: /examples/tree/main/color-modes
+    - name: Bootstrap with React
+      description: "Import and bundle Bootstrap's source Sass and JavaScript with React, Next.js, and React Bootstrap."
+      url: /examples/tree/main/react-nextjs
+      indexPath: src/pages/index.tsx
+    - name: Bootstrap with Vue
+      description: "Import and bundle Bootstrap's source Sass and JavaScript with Vue and Vite."
+      url: /examples/tree/main/vue
```

Comment:
Maybe we should sort these a little different? I feel the current sorting is a little weird, but I don't know if sorting by name in the template would make sense. Maybe a manual sorting would be better?

Reply:
Yeah I didn't understand the original order when I added the new examples so I've put them in the end. I agree that it can be improved but IDK know how exactly. If we don't find any idea or logic to sort them, maybe the right thing to do is to do it alphabetically.
If you have any ideas, you can commit them directly. I'll try to think about it too.

Reply:
Better IMO. Maybe we can also remove the "Bootstrap with " prefixes. Not sure there are useful for readability.

Reply:
Yeah, I was about to drop them

Reply:
Oh yeah or putting it everywhere like you did. One or the other no problem :p

Reply:
I kept the same sorting we have in https://github.com/twbs/examples#readme

Reply:
Good idea. LGTM 👌